.github/workflows/prepare-release.yml

```diff
@@ -16,9 +21,10 @@ jobs:
 
       - name: Update Changelog
         id: update-changelog
-        uses: realm/ci-actions/update-changelog@92b1c91f266809c4374947b8846c09c9049b6f3d
+        uses: realm/ci-actions/update-changelog@21e5e3c4acab316ccdeab748334c9c194d327fb7
```

Comment:
What is the reason of the change in [realm/ci-actions/update-changelog@21e5e3c4acab316ccdeab748334c9c194d327fb7](https://github.com/realm/ci-actions/blob/21e5e3c4acab316ccdeab748334c9c194d327fb7/update-changelog/src/helpers.ts#L118) ?

Reply:
It's just picking up the latest version from main. The reason is that the ability to specify the version from the outside was added in a recent commit.